src/stores/SettingsStore.ts

```diff
@@ -31,35 +42,168 @@ const defaultValues: SettingsType = {
   unifiedCommentsView: false,
 };
 
+const LEGACY_SETTINGS_KEY = 'OSIM::USER-SETTINGS';
```

Comment:
This key is also used for the new config, I don't think it should be named `LEGACY`, also later is used without the constant: https://github.com/RedHatProductSecurity/osim/blob/782c7cffaf07b680db008d05762412b6ef9d7fa8/src/stores/SettingsStore.ts#L113

We could just create a `const SETTINGS_KEY = 'OSIM::USER-SETTINGS';` and use it in the migration function and the rest of the code